src/app/bitstream-page/clarin-license-agreement-page/clarin-license-agreement-page.component.ts

```diff
@@ -394,6 +389,14 @@ export class ClarinLicenseAgreementPageComponent implements OnInit {
       });
   }
 
+  private loadSeznamDatasetLicenseContent() {
+    this.item$.subscribe((item$) => {
+      if (this.item$.value.firstMetadataValue('dc.rights') === this.LICENSE_NAME_SEZNAM) {
```

Comment:
`$` is used for observable/async objects, when you subscribe some object you are accessing its value like `this.item$.value` 

it should look like:
```
this.item$.subscribe((item) => {
      if (item.firstMetadataValue('dc.rights') === this.LICENSE_NAME_SEZNAM) ...
```